core/domain/services/assets/CoreAssetManager.php

```diff
@@ -433,7 +433,7 @@ private function loadAccountingJs()
         // @link http://josscrowcroft.github.io/accounting.js/
         $this->addJavascript(
             CoreAssetManager::JS_HANDLE_ACCOUNTING_CORE,
-            EE_THIRD_PARTY_URL . 'accounting/accounting.js',
+            EE_GLOBAL_ASSETS_URL . 'accounting/accounting.js',
```

Comment:
@tn3rb Should I include the minified version here?

Reply:
oh ya, absolutely, if we have it, thnx